introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-falcon.ipynb

```diff
@@ -439,6 +439,48 @@
     "***"
```

Comment:
<p>secs -&gt; seconds</p><p>response time limit of 60s: can we add a reference to documentation on this?</p><p>input output tokens -&gt; input and output tokens</p><p>20000 -&gt; 2000 ?</p><p>note that these limits are when the inference parameters such as num_beams are set to default: delete this sentence</p><p>Setting non-default environment variables: should we note this can only be done from the SDK currently?</p><p>If using g5.48x -&gt; For example, if using ml.g5.48xlarge</p><p>interval, endpoint -&gt; interval, the endpoint</p><p>can not handle the same thresholds as above. + This is because the TGI container batches requests. While this results in significant throughput improvements, it may cause CUDA OOM or timeout errors.</p><p>once endpoint suffers OOM CUDA error, endpoint -&gt; once an endpoint suffers a CUDA OOM error, the endpoint</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>2000 is correct.  Addressed the reset. Do not have public document which states the limit is 60s and can not be increased. There is only a github issue I am aware of and don't want to cite that here https://github.com/aws/sagemaker-python-sdk/issues/1119</p>